.github/workflows/pr_build.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
       shell: pwsh
       run: |
         $filePath = "src\RestClientVS\source.extension.cs"
-        $replacement = 'Version = "${{ env.VSIX_VERSION }}"'
+        $replacement = 'Version = "${{ steps.vsix_version.outputs.version-number }}"'
```

Comment:
I wonder if this thing can be moved out to a component as well, so you just have to specify the file path

Reply:
Lemme look at this one -- this is a very "mads-specific" step :-) - so can be easy to ensapsulate in the previous composite in one go (same value, just updating two places).  I noticed that some of your other extensions don't follow this pattern so that's why i kept it separate...but can merge.

Reply:
Everyone using my extension .ps1 script follows the same pattern. AFAIK, the only variation is where the `source.extension.cs` file is located which can change when the folder structure is different in some cases.